app/controllers/articleController.js

```diff
@@ -1,7 +1,75 @@
+dotenvConfig(); // load .env variables
+import { config as dotenvConfig } from "dotenv";
+import ArticleAccessor from "../databaseAccessors/articleAccessor.js";
+import UserAccessor from "../databaseAccessors/userAccessor.js";
+import bcrypt from "bcryptjs"; // import bcrypt to hash passwords
+import jwt from "jsonwebtoken"; // import jwt to sign tokens
+import Authorize from "../auth/authorization.js";
+import { ArticleUpdate, ArticleResponse } from "../models/apiModels/article.js";
+import {
+  ErrorInvalidArticleStatus,
+  ErrorInvalidArticleAuthors
+} from "../error/httpErrors.js";
+
 /**
  * ArticleController Class
  *
  * This class controls the behaviour of any web request
  * related to Articles.
  */
-export default class ArticleController {}
+export default class ArticleController {
+  /**
+   * updateStatus method
+   *
+   * Handles the request to update the status of an article.
+   *
+   * @param {Request} req
+   * @param {Response} res
+   */
+  static async updateStatus(req, res) {
+    try {
+      const { slug } = req.params;
+
+      const updates = new ArticleUpdate(req.body);
+
+      const updatedArticleData = await ArticleAccessor.updateArticle(slug, updates);
+
+      // Validate and construct an ArticleResponse instance
+      const updatedArticleResponse = new ArticleResponse(updatedArticleData.toObject());
+
+      // Send the validated ArticleResponse
+      res.status(200).json(updatedArticleResponse);
+    } catch (e) {
+      ErrorInvalidArticleStatus.throwHttp(req, res);
```

Comment:
This is not the only error this method can catch and throw. 

`ArticleAccessor.updateArticle` can throw an internal database error, or an unexpected error. 

Best to catch them here so we know in the call stack what exact error occurred